src/doublyLinkedList.js

```diff
@@ -0,0 +1,110 @@
+class Node {
+
+  constructor( data ) {
+    this._data = data
+    this._prev = null
+    this._next = null
+  }
+
+  data() {
+    return this._data
+  }
+
+  prev() {
+  	return this._prev
+  }
+
+  next() {
+    return this._next
+  }
+
+}
```

Comment:
not quite sure what is going on with data() , prev() and next(). I dont remember these functions being part of the spec for this data structure. Would love to know more about how this works!